.env.sample

```diff
@@ -72,7 +72,11 @@ DOWNLOADABLE_URL_EXPIRY_IN_SECONDS = 300
 # GOTENBERG_URL=http://localhost:3000                                            // This line sets the URL for the local Gotenberg service to `http://localhost:3000'
 
 #TIMEZONE DIFFERENCE VALUE
-TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC= +05:30 
+TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC = +05:30 
 
 #API doc
-API_DOC_URL= "/project/api-doc"
\ No newline at end of file
+API_DOC_URL= "/project/api-doc"
+
+INTERFACE_SERVICE_URL = "http://localhost:9000"                                  // Interface service base url
+USER_SERVICE_NAME = "/user"                                                      // User service name
```

Comment:
@Prajwal17Tunerlabs can we make thisUSER_SERVICE_BASE_URL please

Reply:
I have changed the env variable name to the proposed one.